docs/user/getting-started.md

```diff
@@ -222,29 +222,137 @@ Browse to `ui.flightctl.MY.DOMAIN` and login with the demouser obtained from the
 
 ## Building a Bootable Container Image including the Flight Control Agent
 
-Next, we will use [Podman](https://github.com/containers/podman) to build a [bootable container image (bootc)](https://containers.github.io/bootc/) that includes the Flight Control Agent binary and configuration. The configuration contains the connection details and credentials required by the agent to discover the service and send an enrollment request to the service.
+Next, we will use [Podman](https://github.com/containers/podman) to build a [bootable container image (bootc)](https://containers.github.io/bootc/) that includes the Flight Control Agent binary and configuration. The configuration contains the connection details and credentials required by the agent to discover the service and send an enrollment request to the service. After deploying the flightctl server, this configuration can be retrieved after a Certificate Signing Request has been completed as outlined in the steps below.
 
-Retrieve the agent configuration with enrollment credentials by running:
+### Create Certificate Signing Request configuration
+
+For enrollment, the device will need an enrollment certificate to be present. The enrollment certificate is created when a certificate signing request in flightctl is approved by an authorized user and can be retrieved once this process is complete. The four steps below describe this process.
+
+#### 1/4 Create the signed CSR
+
+This CSR will be embedded in the CSR resource configuration file that will be applied to flightctl.
+
+Create a certificate signing request `.csr` file with openssl:
 
 ```console
-$ flightctl certificate request --scope=enrollment --validity=1y -o agent-config > config.yaml
+openssl req -new -sha256 -key myeckey.pem -out mycsr.csr
+```
 
-[...]
+**NOTES**:
+
+1. The signing key passed with `-key` MUST be an ECDSA key. To generate an ECDSA key, use `openssl ecparam -name secp521r1 -genkey -noout -out myeckey.pem` or see the [openssl documentation on ECDSA](https://docs.openssl.org/1.0.2/man1/ecparam/#synopsis) for more options.
+2. The Subject Common Name in the CSR MUST be either blank or at least 16 characters in length.
+
+For more options, including generating a new private key to sign the CSR, see [openssl's documentation](https://docs.openssl.org/master/man1/openssl-req/#options).
+
+#### 2/4 Create the CSR resource configuration file
+
+You can create a CSR resource configuration file wrapping the CSR file above either by generating it with flightctl or manually. Each option is described below. You may choose the CSR name, and desired expiration in seconds.
+
+##### Option 1: Generate the CSR config file
+
+Issue the command below, specifying your `.csr` file and optionally specifying the CSR name, expiration in seconds, and an outfile. The CSR name defaults to `mycsr`, the expiration to 604800 seconds, and the outfile to `mygeneratedcsr.yaml`.
```

Comment:
```suggestion
Issue the command below, specifying your `.csr` file and optionally specifying the CSR name, expiration in seconds, and an outfile. The CSR name defaults to `mycsr`, the expiration to 604800 seconds, and the output file to `mygeneratedcsr.yaml`.
```

Reply:
I don't think this example with default values is necessary.  Just show a typical example and let them refer to --help for more information.